keras_nlp/models/albert/albert_classifier.py

```diff
@@ -107,19 +66,22 @@ class AlbertClassifier(Task):
         sequence_length=128,
     )
 
-    # Create a AlbertClassifier and fit your data.
+    #Pretrained classifier.
     classifier = keras_nlp.models.AlbertClassifier.from_preset(
         "albert_base_en_uncased",
         num_classes=4,
         preprocessor=preprocessor,
     )
+    # Re-compile (e.g., with a new learning rate).
```

Comment:
I think you are missing in the bert classifier example where we run fit before  "re compiling"